Tests/WhoopDIKitTests/Module/TestModules.swift

```diff
@@ -49,7 +49,11 @@ class NilSingletonModule: DependencyModule {
 
 protocol Dependency { }
 
-class DependencyA: Dependency { }
+class DependencyA: Dependency, Equatable {
```

Comment:
Any reason we don't make this a struct and get that for free?

Reply:
💡 - a good one